### PR TITLE
Bump to ruby 3.3.0

### PR DESCRIPTION
**Pull request by fsanggang**
### 📝 Description

There are some `rubocop`-related changes also, so this might be easier to review commit by commit. (But honestly, they are mostly just ignoring the same rules we already ignore in Investapp).

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206747143994466

### CODE REVIEW

File: flappi.gemspec [hunk 1]
```diff
@@ -14,11 +14,11 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'activesupport', '>4.2.7'
   s.add_runtime_dependency 'recursive-open-struct'
 
-  s.add_development_dependency 'maxitest'
+  s.add_development_dependency 'maxitest', '~> 4'
```
Comment (alto) [inline flappi.gemspec:17]: Yep, fine for now. 👍🏻

File: lib/flappi/optional/flappi.rake [hunk 2]
```diff
@@ -13,7 +13,7 @@ namespace :flappi do
 
       target_paths.each do |document_version, to_path|
         paths = Flappi.configuration.definition_paths[document_version]
-        paths = paths.is_a?(Array) ? paths : [paths]
+        paths = [paths] unless paths.is_a?(Array)
```
Comment (alto) [inline lib/flappi/optional/flappi.rake:16]: I would have thought that

```ruby
paths = Array(paths)
```

does exactly that. 🤔

**Review by alto — APPROVED** [review]
LGTM

File: lib/flappi/optional/flappi.rake (hunk 2 above)
Comment (fsanggang) [reply lib/flappi/optional/flappi.rake]: Yes, what you have works too and it's more succinct! Updated in [763e16c](https://github.com/sharesight/flappi/pull/63/commits/763e16c33e8ab94187ab86d4b197e7ed9b38d2a7)—thanks!

**Pull request merged**